2d/pong_mcru/level/level.gd

```diff
@@ -0,0 +1,46 @@
+extends Node
+
+var player_score = 0
+var opponent_score = 0
+
+
+func score_achieved(score):
+	"""
+	Stops the ball, starts the count down and sums one into the score.
+	"""
+	get_tree().call_group("BallGroup", "stop_ball")
+	$CountDownLabel.visible = true
+	$CountDownTimer.start()
+	return score + 1
+
+
+func _on_PlayerGoal_body_entered(body):
+	"""
+	Checks if Opponent makes point
+	"""
+	opponent_score = score_achieved(opponent_score)
+
+
+func _on_OpponentGoal_body_entered(body):
+	"""
+	Checks if Player makes point
+	"""
+	player_score = score_achieved(player_score)
+
+
+func _process(delta):
```

Comment:
Why do you use here `_process()` instead of `_physics_process()` as in the ball and player code? I think the score board should be consistent with the rest of the game. 

AFAIU both `_process()` and `_physics_process()` are executed every frame, but `_physics_process()` is executed in regular delta steps.

here there is an article about this. https://godotengine.org/qa/38365/what-the-difference-between-_physicsprocess-and-_process

Reply:
I think use `_physics_process()` is not necessary because the refresh of the scores doesn't need be update on each delta step, right? It doesn't need to be accurate with physic processes 

I have tried changing `_process()` by `_physics_process()` but there isn't any difference. Maybe the most efficient is to use the function that is executed less times or unit of time